src/lib/libwebgl.js

```diff
@@ -3205,7 +3205,7 @@ for (/**@suppress{duplicate}*/var i = 0; i <= {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
 #if GL_DEBUG
       console.dir(match);
 #endif
-      explicitUniformLocations[match[5]] = jstoi_q(match[1]);
+      explicitUniformLocations[match[5]] = Number(match[1]);
```

Comment:
Hmm, is the 5th capture not `\w+`, which accepts more than digits? Am I computing that wrong, or perhaps does the GL API guarantee an integer here?

Reply:
We are using the first capture here aren't we?   That looks like `(-?\d+)` to me

Reply:
Oh, somehow my eye read the `match[5]` on the left...